python/example_code/iotthingsgraph/cloudcamera.py

```diff
@@ -0,0 +1,144 @@
+# specific language governing permissions and limitations under the License.
+#
+
+# snippet-sourcedescription:[cloudcamera.py demonstrates how to mock an AWS IoT Things Graph camera device in the cloud.]
+# snippet-service:[iotthingsgraph]
+# snippet-keyword:[Python]
+# snippet-keyword:[AWS IoT Things Graph]
+# snippet-keyword:[Code Sample]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2019-07-25]
+# snippet-sourceauthor:[AWS]
+# snippet-start:[iotthingsgraph.python.cloudcamera.complete]
+
+from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTClient
+import logging
+import time
+import argparse
+import json
+
+AllowedActions = ['both', 'publish', 'subscribe']
+
+# Custom MQTT message callback
+def customCallback(client, userdata, message):
+    print("Received a new message: ")
+    print(message.payload)
+    print("from topic: ")
+    print(message.topic)
+    print("--------------\n\n")
+
+
+# Read in command-line parameters
+parser = argparse.ArgumentParser()
+parser.add_argument("-e", "--endpoint", action="store", required=True, dest="host", help="Your AWS IoT custom endpoint")
+parser.add_argument("-r", "--rootCA", action="store", required=True, dest="rootCAPath", help="Root CA file path")
+parser.add_argument("-c", "--cert", action="store", dest="certificatePath", help="Certificate file path")
+parser.add_argument("-k", "--key", action="store", dest="privateKeyPath", help="Private key file path")
+parser.add_argument("-p", "--port", action="store", dest="port", type=int, help="Port number override")
+parser.add_argument("-w", "--websocket", action="store_true", dest="useWebsocket", default=False,
+                    help="Use MQTT over WebSocket")
+parser.add_argument("-id", "--clientId", action="store", dest="clientId", default="basicPubSub",
+                    help="Targeted client id")
+parser.add_argument("-t", "--topic", action="store", dest="topic", default="Camera1/capture/finished", help="Targeted topic")
+parser.add_argument("-m", "--mode", action="store", dest="mode", default="both",
+                    help="Operation modes: %s"%str(AllowedActions))
+parser.add_argument("-M", "--message", action="store", dest="message", default=True,
+                    help="Message to publish")
+parser.add_argument("-n", "--thingName", action="store", dest="thingName", default="Bot", help="Targeted thing name")
+
+args = parser.parse_args()
+host = args.host
+rootCAPath = args.rootCAPath
+certificatePath = args.certificatePath
+privateKeyPath = args.privateKeyPath
+port = args.port
+useWebsocket = args.useWebsocket
+clientId = args.clientId
+topic = args.topic
+thingName = args.thingName
+
+images = ["https://images-na.ssl-images-amazon.com/images/I/41iz5Tw82IL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/51rMLSWgwRL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/31s6UyPtjOL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/41+K4pC74XL._AC_US218_.jpg"]

```

Comment:
This line is 325 chars long. Can you put each item in the list on a separate line?